src/pages/index.js

```diff
@@ -78,24 +126,123 @@ addcreateCardButton.addEventListener("click", () => {
   formValidators["add-button-form"].resetValidation();
 });
 
+avatarImage.addEventListener("click", () => {
+  avatarEditModal.open();
+});
+
 // EVENT HANDLERS
 
+function handleLikeCard(cardId) {
+  return api
+    .likeCard(cardId)
+    .then(() => {
+      const card = cardsSection.getItem(cardId);
+      if (card) {
+        card._updateLikeStatus(true);
```

Comment:
`_updateLikeStatus`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.

**Please, correct it everywhere**